protocols/cifs/src/main/java/org/opennms/netmgt/poller/monitors/JCifsMonitor.java

```diff
@@ -216,6 +223,32 @@ public boolean accept(SmbFile smbFile, String s) throws SmbException {
         return serviceStatus;
     }
 
+    private void setFinalStatic(final Field finalStaticField, final Object value) throws NoSuchFieldException, IllegalAccessException {
```

Comment:
Can we name this something like `setFieldValue(...)` or something, as this works for all fields, not only static ones.

Reply:
done